_posts/2018-04-01-from-chemical-engineering-to-software-development.md

```diff
@@ -0,0 +1,114 @@
+
+Unfortunately, I give here only the differences between a non-tech industry and the software industry, therefore, the purpose of this article is not about how to start your software engineer career (however, you can still drop me a message if you are looking for any pieces of advice).
+
+The differences between the British and the French way of working are not tackled here, it is a wide subject deserving a full and proper article (trailer: I am writing from London, which gives you a good idea of my humble opinion).
+
+## Where it all began...
+
+With few words, the `Once upon a time` is: I grew up on a French island ([Reunion Island](https://www.google.co.uk/maps/place/Reunion/@-21.1351121,55.2471124,10z/data=!3m1!4b1!4m5!3m4!1s0x2178778110b8e43b:0x4a7f8e89ecdbeaf9!8m2!3d-21.115141!4d55.536384) - oversea territory), did a normal scholarship until my _baccalauréat_ (UK GCSE equivalent), attended to intensive courses in mathematics, physics and chemistry for 2 years (_Classe Preparatoire aux Grandes Ecoles_), was admitted into a national chemical engineering school ([ENSIC](https://ensic.univ-lorraine.fr/)) in the lovely town of Nancy (the Lorraine region, in the mainland), graduated after 3 years and found my first job as a chemical engineer in Paris.
+
+From that point, the career path was almost sealed: I would have spent the next decades in the same company (because it shows loyalty and mind diversity is not really good you know?), painfully risen through the ranks, fought against this silly bureaucracy where more senior people teach you what to do in a world they can no longer understand, gone in retirement at 70 years old with nothing (stay lucid, this is going to happen).
+
+Fortunately enough, I did not spend decades where I started. After only 2 years and a half in Paris, doing the same repetitive and boring tasks, under an awful and stressful management, I was on my knees, dreamless and buried below this dull routine under morphine.
+
+After a lot of discussions with my cousin [Tom](https://github.com/ostapneko), his wife [Emma](https://github.com/emma-p), and her brother [Alex](https://github.com/onima) (all of them are developers), I decided to make the move. I will be forever thankful to them for having the patience and taking the time to onboard me in this amazing industry.
+Like I said few lines above, I won't give any tips to set someone on the right tracks to become a software engineer (perhaps another article?), but I will present the main contrasts between my former job and my current one.
+
+## Timescale and feedback loop are not the same
+
+Let's start with the most obvious one: velocity of execution is nothing but comparable.
```

Comment:
this could be more simply written with just: `Let's start with the most obvious one: velocity of execution.`